Bottom: 37e7ccc382017c5f5e221e651c5ece331c8757be
Top:    cdc226230e45430773c967a5b19e606c9ac5c26c
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-16 04:11:37 -0300

Refresh of manual-document-mt-safety-filesys.patch

---

diff --git a/manual/filesys.texi b/manual/filesys.texi
index 47176cc..b5b6630 100644
--- a/manual/filesys.texi
+++ b/manual/filesys.texi
@@ -3523,7 +3523,7 @@ The @code{mkdtemp} function comes from OpenBSD.
 @c fchmodat
 @c fchownat
 @c futimesat
-@c fstatat
+@c fstatat (there's a commented-out safety assessment for this one)
 @c linkat
 @c mkdirat
 @c mkfifoat
